build/yamls/antrea-ipsec.yml

```diff
@@ -490,6 +490,14 @@ spec:
           periodSeconds: 10
           timeoutSeconds: 5
         name: antrea-agent
+        ports:
+        - containerPort: 8080
+        readinessProbe:
+          httpGet:
+            path: /server/healthz
+            port: 8080
```

Comment:
The opened port is set when starting apiserver, you can check code and find the port. The path is wrong too, you can curl the port to see the correct path